lib/utils/setCookie.js

```diff
@@ -42,7 +46,7 @@ module.exports = function setCookie (storage, name, value) {
     } else {
       for (; i < ll; i++) {
         clone.domain = domains[i]
-        cookie.setItem(name, value, expires, clone.path, clone.domain)
+        cookie.setItem(name, value, expires, clone.path, clone.domain, true, sameSite)
```

Comment:
IMO, for easier read:
`cookie.setItem(name, value, expires, clone.path, true, 'None')`